lib/cache/cache_base.js

```diff
@@ -32,7 +31,7 @@ class CacheBase extends EventEmitter {
     }
 
     get _options() {
-        const opts = config.get(this._optionsPath);
+        const opts = require('config').get(this._optionsPath);
```

Comment:
Yes it's because the config data is loaded and cached after the very first require('config'), and the config module doesn't provide any API for updating a loaded config in memory. In the `handleCommandLine()` method in unity_cache_server.js, we need to set environment overrides before the config is parsed, so this change is to make sure no module dependencies are prematurely parsing and caching the config.